charts/substra-backend/templates/rbac.yaml

```diff
@@ -25,7 +25,13 @@ rules:
   - apiGroups: [""]
     resources: ["secrets"]
     verbs: ["get", "watch", "list", "create", "update", "patch"]
----
+  - apiGroups: [""]
+    resources: ["pods"]
+    verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
+  - apiGroups: ["batch", "extensions"]
+    resources: ["jobs"]
+    verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
+  ---
 kind: RoleBinding
```

Comment:
@AurelienGasser Can you confirm if this rbac is enough ?

Reply:
@Kelvin-M I get this in the worker. The wrong service account is getting used apparently
```
│ HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"jobs.batch  │
│ is forbidden: User \"system:serviceaccount:org-1:default\" cannot create resource \"jobs\" in API group \"batc │
│ h\" in the namespace \"org-1\"","reason":"Forbidden","details":{"group":"batch","kind":"jobs"},"code":403}
```